tasks/apply-mapping/README.md

```diff
@@ -25,6 +25,13 @@ This task supports variable expansion in tag values from the mapping. The curren
 | dataPath | Path to the JSON string of the merged data to use in the data workspace | No | |
 | failOnEmptyResult | Fail the task if the resulting snapshot contains zero components | Yes | false |
 
+## Changes in 1.5.0
+* Repository Handling:
```

Comment:
I'd suggest to have a full sentence and also maybe make it clearer what repository we're talking about here, like:
```suggestion
* Change the way the component repository field in RPA mapping is handled
```

Reply:
Done, thanks